components/org.wso2.carbon.identity.sts.passive/src/main/java/org/wso2/carbon/identity/sts/passive/custom/provider/CustomAttributeProvider.java

```diff
@@ -71,14 +72,20 @@ private AttributeBean createAttributeFromClaim(ProcessedClaim claim, String toke
         AttributeBean attributeBean = new AttributeBean();
         if (WSS4JConstants.WSS_SAML2_TOKEN_TYPE.equals(tokenType)
                 || WSS4JConstants.SAML2_NS.equals(tokenType)) {
-            attributeBean.setQualifiedName(claim.getClaimType());
             attributeBean.setNameFormat(claim.getClaimType());
-        } else {
-            attributeBean.setQualifiedName(claim.getClaimType());
         }
+        String attributeName = claim.getClaimType();
+        String attributeNamespace = claim.getClaimType();
+
+        if (StringUtils.isNotBlank(attributeNamespace) && attributeNamespace.contains("/") &&
+                attributeNamespace.length() > attributeNamespace.lastIndexOf("/") + 1) {
```

Comment:
can't we use StringUtils endsWith here to check if the last character is not "/"? will make the code more readable.

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-sts/pull/137